src/api/protectedApiClient.ts

```diff
@@ -37,6 +41,11 @@ const changePassword = (request: {
     .catch((e) => e);
 };
 
+const getMyEvents = (): Promise<EventInformation[]> => {
+  return AppAxiosInstance.get(ProtectedApiClientRoutes.MY_EVENTS).then(
+    (r) => r.data?.events,
```

Comment:
I don't think we need an optional chain here - if the response is successful, it will contain the events field.

Reply:
for readability: 🔧 `r` => `res`